crates/wasm/wit/entity.wit

```diff
@@ -24,6 +24,22 @@ default interface entity {
         apply-base-pose: bool,
     }
 
+    /// Animation clip data
```

Comment:
Hm might be a bit more rust-like to do `result<animation-clip, animation-error>`.
`enum animation-error { not-loaded, failed-loading(string) }` (not sure what the exact syntax is).

Also naming: `animation-clip` sounds like it's the actual clip. Maybe `animation-clip-metadata` or something (and same for `get-animations-clips`, since it's not returning the actual clip but just metadata).

Reply:
Added status and moved the api to asset.wit
```
    /// Cache status of an asset
    variant asset-cache-status {
        /// Ready to be used
        ready,
        /// Not yet loaded
        not-loaded,
        /// Asset has an error
        failed-loading(string)
    }
    /// Animation asset metdata
    record animation-asset-metadata {
        /// BinderId for the animation tracks
        binders: list<string>,
        /// Duration in seconds
        duration: float32,
        /// Cache status of an animation asset
        status: asset-cache-status,
    }

    get-animation-asset-status: func(clip-url: string) -> asset-cache-status
    get-animation-asset-metadata: func(clips: list<string>) -> list<animation-asset-metadata>
```